lib/presentation/pages/intro_page.dart

```diff
@@ -22,7 +24,16 @@ class _IntroPageState extends State<IntroPage> {
   IntroState state = IntroState.welcome;
   final Duration selectionTransitionDelay = const Duration(milliseconds: 200);
 
-  void nextPage() => _introKey.currentState?.next();
+   Future<void> _vibrateDevice() async {
+    if (await Vibrate.canVibrate) {
+      Vibrate.vibrate();
+    }
```

Comment:
We are using controllers for api calls.
Use     `VibrationController.instance.vibrate(VibrationType.light);` directly